sdk/storage/src/datalake/path_client.cpp

```diff
@@ -33,10 +33,11 @@ namespace Azure { namespace Storage { namespace DataLake {
       return std::string(begin, end);
     }
 
-    std::pair<int64_t, int64_t> GetOffsetLength(const std::string& rangeString)
+    std::pair<int64_t, Azure::Core::Nullable<int64_t>> GetOffsetLength(
```

Comment:
When will this pair.second be null?

Reply:
When the range string is in a format of `"bytes=1024-"`.

Reply:
Can you name one API that would return this in response?

Reply:
I think you are correct, this method is used for parsing response so we shouldn't be considering the request's format.
Changed in latest commit.